### ISSUE TITLE
WARNING [supervisor.addons.validate] Add-on config 'startup' with 'before' is deprecated

### ISSUE THREAD

**Issue by crgarcia12** [imthemichi/hassio-flashcc2531#2]
```
21-04-11 17:08:44 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/imthemichi/hassio-flashcc2531 repository
21-04-11 17:08:44 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/danielwelch/hassio-zigbee2mqtt repository
21-04-11 17:08:44 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/hassio-addons/repository repository
21-04-11 17:08:44 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/home-assistant/addons repository
21-04-11 17:08:45 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'startup' with 'before' is deprecated. Please report this to the maintainer of flashcc2531
21-04-11 17:08:45 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'devices' use a deprecated format, the new format uses a list of paths only. Please report this to the maintainer of flashcc2531
21-04-11 17:08:45 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'startup' with 'before' is deprecated. Please report this to the maintainer of flashcc2531
21-04-11 17:08:45 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'devices' use a deprecated format, the new format uses a list of paths only. Please report this to the maintainer of flashcc2531
```

**Comment by imthemichi** [comment]
it seems, hassio changed the config.json. before should now be services.  https://developers.home-assistant.io/docs/add-ons/configuration

**Comment by imthemichi** [comment]
or maybe it should be once...

**Comment by imthemichi** [comment]
devices should change from

"devices": ["/dev/mem:/dev/mem:rw", "/dev/gpiomem:/dev/gpiomem:rw"],

maybe to

"devices": ["/dev/mem:rw", "/dev/gpiomem:rw"],

**Comment by imthemichi** [comment]
Issue is closed with the last commit c2f3755 on master
